outputs.tf

```diff
@@ -15,7 +15,7 @@ output "mgmt_port" {
 
 # BIG-IP Password
 output "password" {
-  value = ""
+  value = "${random_password.bigippassword.result}"
```

Comment:
value = "${random_password.bigip_password.result}"

Reply:
I've committed a change to the pull request which addresses your request